doc/user/content/release-notes.md

```diff
@@ -48,6 +48,10 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.7.2 %}}
 
+- Record Kafka Consumer metrics in the `mz_kafka_consumer_statistics` system
```

Comment:
nit: instead of the if clause I would probably just point to the docs for this with option

Reply:
Sounds good, thanks!

Reply:
I looked at doing this and, with our current docs, there is no single place to point to for this option. We have different pages for each of the `text`, `bytes`, `avro` formats.

Reply:
Also, I'm going to make this "on by default" in another commit, so we may not even need to mention the `WITH` option.

Reply:
i would just point to the avro page which i think folks have done in the past? i think i did that once for a with option

Reply:
Sure!

Reply:
I decided to remove the mention of this variable, as we're going to be enabling these metrics by default.